### PR TITLE
Compatability with Python3

### LINKED ISSUE #2: Python 3 support
Any chance of Python 3 support?

### PR DESCRIPTION
**Pull request by dave2**
This makes the extension run with python3, at least as far as I tested.

YMMV.

### CODE REVIEW

File: sphinxcontrib/aafig.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ class AafigDirective(images.Image):
     def run(self):
         aafig_options = dict()
         image_attrs = dict()
-        own_options_keys = self.own_option_spec.keys() + ['scale']
+        own_options_keys = list(self.own_option_spec.keys()).append('scale')
```
Comment (llucax) [inline sphinxcontrib/aafig.py]: Two issues here, first I don't see why the original line won't work in Python3. It should.
The seconds and more important is `list(self.own_option_spec.keys()).append('scale')` will **always** return `None`.

**Review by llucax — CHANGES_REQUESTED** [review]
All changes but the one with a comment look good to me. But the one with a comment looks broken, unless I'm missing something...

**Comment by llucax** [comment]
Thanks for the PR!

**Comment by llucax** [comment]
There is also a typo in the commit message "Compatability" (should be "Compatibility") :relaxed:

File: sphinxcontrib/aafig.py (hunk 1 above)
Comment (dave2) [reply sphinxcontrib/aafig.py]: `keys()` on a `dict` in py3 returns type `dict_keys`, which is read-only, and doesn't have an append. Needs to be converted to a list to have an append.  For example:

```
>>> x = {"abc": "efg"}
>>> x.keys() + ['scale']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'dict_keys' and 'list'
```
However, yes it always returns `None` as written, it should be `list(x.keys())+['scale']`. I'll fix it.

Comment (llucax) [reply sphinxcontrib/aafig.py]: > `keys()` on a `dict` in py3 returns type `dict_keys`, which is read-only

Right! I missed the `.keys()`. The problem is not that it is read-only, but that it's a generator, and like the error says, it doesn't define a `+` operator with a list.

> However, yes it always returns `None` as written, it should be `list(x.keys())+['scale']`. I'll fix it.

Yeah, perfect, that looks good to me!

**Review by llucax — APPROVED** [review]

**Comment by llucax** [comment]
If it's not much of a trouble, I'd prefer if you squash both commits and fix the typo in the first one. If it's too much bother, I can merge as it is also, not a big deal.

**Comment by dave2** [comment]
Ah yeah give me a moment, I'll squash them now they're good.

**Comment by llucax** [comment]
Thanks!

**Pull request merged**